packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/destinationConnector-XCA Endpoint-responseTransformer-step-1-Case 1 - Success or PartialSuccess.js

```diff
@@ -50,8 +49,7 @@ if ('Success' == queryResponseCode.toString() || 'PartialSuccess' == queryRespon
 				if (newDocumentUniqueId) attachment.newDocumentUniqueId = newDocumentUniqueId.toString();
 
 				const documentEncoded = entry.*::Document;
-				const documentDecoded = decodeBase64(documentEncoded.toString());
-				const parsedFile = parseFileFromString(documentDecoded.toString());
```

Comment:
file being converted to a string then decoded to base64 and then converted to a string again, before getting passed to function parseFileFromString which tried to properly decode base64.